### PR TITLE
Add flag `skip-existing` to skip upload if release exists

### LINKED ISSUE #70: Feature: Support appending an existing release
**Problem**
If a release already exists with the same tag then chart-releaser fails with the following error:

```Error: error creating GitHub release: POST https://api.github.com/repos/my_org/actions-runner-controller-mirror/releases: 422 Validation Failed [{Resource:Release Field:tag_name Code:already_exists Message:}]```

**Proposed Soltuion**
Check for a matching release beforehand and append to that release if the action is configured to do so. 

```yaml
      - name: Run chart-releaser
        uses: helm/chart-releaser-action@v1.1.0
        env:
          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
          APPEND_TO_RELEASE: true
```

To retain compatibility make the default value false.

### PR DESCRIPTION
**Pull request by xunleii**
closes #70 (following the last comment https://github.com/helm/chart-releaser-action/issues/70#issuecomment-919904216)

### CODE REVIEW

**Comment by szaimen** [comment]
Hi, this should already be possible with https://github.com/helm/chart-releaser/blob/ee26f50ce93ca23920dc1ce93e677665ce65a46e/README.md?plain=1#L156 IIRC...

**Comment by xunleii** [comment]
Hi, you're right @szaimen, I didn't see this var. 
Personnaly, adding it directly as an action parameter seems better for me to keep consistency between action parameters and CR parameters (like `owner` for `CR_OWNER` or `repo` for `CR_GIT_REPO`). What do you think ?

File: action.yml [hunk 1]
```diff
@@ -23,7 +23,10 @@ inputs:
     description: "Just install cr tool"
     required: false
   skip_packaging:
-    description: "skip the packaging option (do your custom packaging before running this action"
+    description: "Skip the packaging option (do your custom packaging before running this action"
```
Comment (cpanato) [inline action.yml:26]:
```suggestion
    description: "Skip the packaging option (do your custom packaging before running this action)"
```

**Review by cpanato — CHANGES_REQUESTED** [review]
need a rebase

**Review by cpanato — APPROVED** [review]
lgtm, not against, but we might don't  want to add all possible due to limitation of the github action inputs

**Pull request merged**